k8s/prod/cluster-01-overlay/kustomization.yaml

```diff
@@ -22,9 +21,6 @@ patchesStrategicMerge:
   #traefik patch
   - ../../admin/traefik/patches/prod/cluster-01/traefik.yaml
 
-  #kured patch
-  - ../../kured/patches/prod/cluster-01/kured.yaml
```

Comment:
doubt this will fix it, I think there's an issue with the node, I've deleted the pod twice  and it's always the same node having the issue: aks-nodepool-19711934-1

Reply:
Ahhk, thought i would do default trouble shoot :D